engines/gob/inter_v1.cpp

```diff
@@ -700,6 +709,13 @@ void Inter_v1::o1_callSub(OpFuncParams &params) {
 		return;
 	}
 
+	// Skipping the copy protection screen in Adibou2
+	if (!_vm->_copyProtection && (_vm->getGameType() == kGameTypeAdibou1) && (offset == 1746) &&
```

Comment:
kGameTypeAdibou1 or should it be 2?

Reply:
It is indeed kGameTypeAdibou1, but the comment above is wrong (should be "Skipping the copy protection screen in Adibou **1**").